app/(dashboard)/dashboard/interviews/_actions/utils.ts

```diff
@@ -0,0 +1,60 @@
+import {
+  caseProperty,
+  codebookHashProperty,
+  protocolName,
+  protocolProperty,
+  sessionExportTimeProperty,
+  sessionFinishTimeProperty,
+  sessionProperty,
+  sessionStartTimeProperty,
+} from '@codaco/shared-consts';
+
+import { type Interview, type Protocol } from '@prisma/client';
+import crypto from 'crypto';
+import { hash } from 'ohash';
+
+/**
+ * Creates an object containing all required session metadata for export
+ * and appends it to the session
+ */
+
+export const formatExportableSessions = (
+  sessions: Interview[],
+  protocols: Protocol[],
+) =>
+  sessions.map((session) => {
+    const sessionProtocol = protocols.find(
+      (protocol) => protocol.id === session.protocolId,
+    );
+
+    if (!sessionProtocol) return;
+
+    const sessionVariables = {
+      [caseProperty]: session.id,
+      [sessionProperty]: session.id,
+      [protocolProperty]: getRemoteProtocolID(sessionProtocol.name),
```

Comment:
This should match `protocol.hash` now. Please delete the `getRemoteProtocolID` function

Reply:
Done ✅